taskcluster/translations_taskgraph/transforms/find_upstreams.py

```diff
@@ -63,7 +63,7 @@ def get_cleaning_type(upstreams):
     candidates = set()
 
     for upstream in upstreams:
-        if upstream.kind not in ("bicleaner", "clean-corpus"):
+        if upstream.kind not in ("clean-corpus", "bicleaner"):
```

Comment:
no reason here, I thought I would need to remove bicleaner here but it broke things, so I added to back. I do not completely understand how this part works

Reply:
This is because `bicleaner` here refers to a `kind` (both `bicleaner` and `bicleaner-ai` are in the `bicleaner` kind`). Basically, this tuple needs to contain any kinds that contain cleaning tasks.